.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,4 +14,4 @@ Fixes #
 - [ ] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
 - [ ] ✅ I've added tests where applicable to test my new functionality.
 - [ ] 📖 I've made sure that my contracts are well-documented.
-- [ ] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).
```

Comment:
Good catch!

Reply:
I caught the mistake in the PR template… because I was obeying the PR template 😀